themes/_core.css

```diff
@@ -451,32 +452,52 @@ body.close .content {
   }
   .github-corner:hover .octo-arm {
     -webkit-animation: none;
-            animation: none;
+    animation: none;
   }
   .github-corner .octo-arm {
     -webkit-animation: octocat-wave 560ms ease-in-out;
-            animation: octocat-wave 560ms ease-in-out;
+    animation: octocat-wave 560ms ease-in-out;
+  }
+}
+
+@media screen and (max-width: 630px) {
+  body.close .sidebar {
+    transform: translateX(0em);
+  }
+  body.close .content {
+    transform: translateX(0rem);
+  }
+  body.close .sidebar-toggle {
+    background-color: rgba(255, 255, 255, 0.8);
+    transition: 1s background-color;
+    display: inline-block;
```

Comment:
So this is where things are a bit tricky: `_core.css` is bundled with Docsify. And it's a big old mess. For this reason I try to avoid changing it and keep my changes isolated in the `cold-tavern` theme.

Do you think we could move some (or all) of this to `cold-tavern/_sidebar.css`?

Reply:
Yeah I think you could move these to the sidebar, just when u do check that the website is reading the css files in order and everything is working the same  as if they were here..I think there is already a 630px @media query in the sidebar so u could just paste it there.